js/main.js

```diff
@@ -4,13 +4,41 @@
 var $car = document.querySelector('.car');
 var $carStyle = $car.style;
 var $windowWidth = window.innerWidth;
+var $windowHeight = window.innerHeight;
 var intervalId = null;
 
 function moveCar() {
-  $carStyle.left = `${data.location.x + 5}px`;
-  data.location.x += 5;
-  if ((data.location.x + 160) > $windowWidth) {
-    clearInterval(intervalId);
+  if (data.currentDirection === 'right') {
+    $carStyle.left = `${data.location.x + 5}px`;
+    data.location.x += 5;
+    if ((data.location.x + 160) > $windowWidth) {
+      clearInterval(intervalId);
+      data.moving = false;
+    }
+  }
+  if (data.currentDirection === 'left') {
+    $carStyle.left = `${data.location.x - 5}px`;
+    data.location.x -= 5;
+    if (data.location.x < 0) {
+      clearInterval(intervalId);
+      data.moving = false;
+    }
+  }
+  if (data.currentDirection === 'up') {
+    $carStyle.top = `${data.location.y - 5}px`;
+    data.location.y -= 5;
+    if (data.location.y < 0) {
+      clearInterval(intervalId);
+      data.moving = false;
+    }
+  }
+  if (data.currentDirection === 'down') {
+    $carStyle.top = `${data.location.y + 5}px`;
+    data.location.y += 5;
+    if ((data.location.y + 160) > $windowHeight) {
+      clearInterval(intervalId);
+      data.moving = false;
+    }
```

Comment:
I would make the work in each of these blocks its own function. You should also make this one set of conditional blocks, not a bunch of individual conditionals.